src/admin/admin.service.ts

```diff
@@ -136,4 +163,44 @@ export class AdminService {
       throw new Error(`Invalid report category: ${value}`);
     }
   }
+
+  async sendTransferRequest(
+    transferReportDto: TransferReportDto,
+  ): Promise<AxiosResponse | null> {
+    let returnValue = null;
+    const data = JSON.stringify({
+      'TL pranešimo ID': transferReportDto.refId,
+      Turinys: transferReportDto.name,
+      Platuma: transferReportDto.latitude.toString(),
+      Ilguma: transferReportDto.longitude.toString(),
+      Statusas: transferReportDto.status,
+      'Data ir laikas': transferReportDto.reportDate.toString(),
+      'Vykdytojo e-mail': transferReportDto.email,
+    });
+
+    const config = {
+      method: 'post',
+      maxBodyLength: Infinity,
+      url: process.env['AADIS_URL'],
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      data: data,
+    };
+
+    await axios
+      .request(config)
+      .then((response) => {
+        if (response.status == 200) {
+          returnValue = response;
+        } else {
+          returnValue = null;
+        }
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
Debugging errors will be challenging since Sentry won't capture these errors, and you'll only find them only in Docker logs. To mitigate this, consider configuring axios `validateStatus` (as explained here: [link](https://stackoverflow.com/questions/43842711/can-i-throw-error-in-axios-post-based-on-response-status)) and consistently return `AxiosResponse` instead of `AxiosResponse|null` from this function. This way, if an error occurs, Sentry will capture it along with supplementary information such as the `transferReportDto` values and you won't need to deal with `null` case.